.circleci/config.yml

```diff
@@ -33,8 +35,56 @@ jobs:
           name: Verify if documentation is up-to-date
           command: npm run test:docs
 
+      - persist_to_workspace:
+          root: ~/
+          paths:
+            - repo
+
+  deploy:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: ~/
+      - run:
+          name: Authenticate with registry
+          command: echo "//registry.npmjs.org/:_authToken=${npmjs_token_cosmo}" > ~/repo/.npmrc
+      - run:
+          name: Publish package
+          command: npm publish
+
+      - run:
+          name: Setup git user
+          command: |
+            git config credential.helper 'cache --timeout=120'
+            git config user.email "cosmo-admin@github.com"
+            git config user.name "cosmo-admin-CircleCI"
+
+      - run:
+          name: Update main branch
+          command: |
+            git checkout ${master_branch}
+            git merge --ff-only ${CIRCLE_BRANCH}
+            git push -q https://${gh_token_qooban}@github.com/cloudify-cosmo/cloudify-ui-common.git ${master_branch}
```

Comment:
gh_token_qooban I don't this is a good idea

Reply:
You mean that it's my token or GitHub token in general?

I talked with Yuval yesterday. After we finish testing he'll switch the tokens to the one from Cloudify-wide account and it'll be possible to merge to master.

For now I used mine and `master_branch` was set to my own branch (not master). Token's value is stored in CircleCI environmental variables and cannot be retrieved directly.